amazon-kinesis-client-multilang/src/test/java/software/amazon/kinesis/multilang/config/KinesisClientLibConfiguratorTest.java

```diff
@@ -306,12 +306,32 @@ public void testWithMissingWorkerId() {
     }
 
     @Test
-    public void testWithMissingStreamName() {
+    public void testWithMissingStreamNameAndMissingStreamArn() {
         thrown.expect(NullPointerException.class);
-        thrown.expectMessage("Stream name is required");
 
-        String test = StringUtils.join(new String[] { "applicationName = b",
-                "AWSCredentialsProvider = " + credentialName1, "workerId = 123", "failoverTimeMillis = 100" }, '\n');
+        String test = StringUtils.join(new String[] {
+                        "applicationName = b",
+                        "AWSCredentialsProvider = " + credentialName1,
+                        "workerId = 123",
+                        "failoverTimeMillis = 100" },
+                '\n');
+        InputStream input = new ByteArrayInputStream(test.getBytes());
+
+        configurator.getConfiguration(input);
```

Comment:
There should be assertions/validations after `getConfiguration`. Otherwise, if the `configurator` silently did nothing this test would naively pass.

Reply:
Doesn't the throw.expect() cover this scenario? If the configurator "did nothing", no exception would be thrown.

Reply:
Ah! You are correct. I recall seeing the `@Test`, and must have overlooked the unmodified `thrown.expect(...)`

Would you kindly change this to `@Test(expected = NPE)`?